.github/workflows/Documentation.yml

```diff
@@ -0,0 +1,50 @@
+name: Documentation
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types:
+      - completed
```

Comment:
Similar question as above. Documentation needs ~11 minutes to build, i.e. waiting time for a PR until everything is green, would increase.

Reply:
See above response.